### DISCUSSION TITLE
Environment Variables Naming Issue

### DISCUSSION THREAD

**Discussion by chandraguptgosavi** [deta/cloud-docs#568] [Help] [answered]
I am getting this error while updating env file.
Error: failed to update env vars: Key JWT_SECRET  has an improper name
The env file looks like this-

:
: 
JWT_SECRET = s@o!m#e^t$h%i&n*g

What am I doing wrong here?

**Comment by el-garro** [comment]
Have you tried surrounding the value with "" ?

**Reply by chandraguptgosavi** [reply]
Yes it doesn't work that way. I got the solution, see the comment section. The weird thing I noticed about this is that was working fine on heroku.

**Comment by chandraguptgosavi** [comment, marked as answer]
It is working after removing spaces adjacent to `=`.
Example
From this -
JWT_SECRET = s@o!m#e^t$h%i&n*g
To this -
JWT_SECRET=s@o!m#e^t$h%i&n*g

**Reply by el-garro** [reply]
Yes I forgot, that's specified in the docs:
https://docs.deta.sh/docs/cli/commands#examples-7
